ddtrace/internal/import_hooks/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .registry import hooks
+from .decorator import register_module_hook
+from .patch import patch, unpatch
+
+__all__ = ['hooks', 'register_module_hook', 'patch', 'unpatch']
+
+# Make sure we always patch
+patch()
```

Comment:
Not sold on this being a good idea or not.

My thinking was if anyone wanted to register an import hook, then we should make sure it will actually work.

But maybe this should be left for a higher level API that actually gets used by the tracer/integrations?

Reply:
Fair question. Maybe call this side-effect higher in the tracer's code? (like at the root of the tracer?). That way side-effects aren't deep hidden in the code / it can likely find a nicer spot there.

Reply:
yeah, I'll move this to `ddtrace/__init__.py`